### PR TITLE
[9.0] partner_firstname - Fix user form with firstname and lastname asking for a mandatory name

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by yajo** [comment]
It would be nice if you can add a test for this.

File: partner_firstname/models/res_user.py [hunk 1]
```diff
@@ -28,3 +28,10 @@ def default_get(self, fields_list):
                 result[field] = inverted.get(field)
 
         return result
+
+    @api.one
```
Comment (leemannd) [inline partner_firstname/models/res_user.py]: deprecated

**Comment by pedrobaeza** [comment]
Please check pylint error

**Comment by yvaucher** [comment]
@pedrobaeza fixed (don't over do, travis is already there to say so :smile: )

@Yajo I just added a test on the onchange of user

**Comment by pedrobaeza** [comment]
:+1: 

@yvaucher, it was just in case you were not aware of it.

**Comment by leemannd** [comment]
:+1:

**Pull request merged**